src/simulation/wire_types/population.proto

```diff
@@ -1,8 +1,9 @@
 syntax = "proto3";
 package population;
 
-message Population {
-  repeated Person persons = 1;
+message Header {
```

Comment:
I didn't really get why we need this header.

Reply:
We don't really need it, I just wanted to see, how one would do something like that. If we actually want to establish a binary file format, having a header with version information and maybe other stuff might be necessary.